app/lib/features/chat_ng/widgets/chat_input/chat_editor.dart

```diff
@@ -139,10 +140,12 @@ class _ChatEditorState extends ConsumerState<ChatEditor> {
     final isKeyboardVisible = ref.watch(keyboardVisibleProvider).valueOrNull;
     final emojiPickerVisible = ref
         .watch(chatInputProvider.select((value) => value.emojiPickerVisible));
+    final isEncrypted =
+        ref.watch(isRoomEncryptedProvider(widget.roomId)).valueOrNull;
```

Comment:
you don't care about it being an option, if I read this right. So why not just go for:
```suggestion
    final isEncrypted =
        ref.watch(isRoomEncryptedProvider(widget.roomId)).valueOrNull == true;
```